guides.md

```diff
@@ -237,6 +239,140 @@ members. Skip the steps below at your discretion for low-stakes topics.
+
+### Setting up employee record and payroll
+
+1. Collect information for [employee record](https://link.hypha.coop/employees).
+
+1. Add Member SIN to Passbolt and share access with `People Operations` group.
+
+1. Collect encrypted (see [Sensitive Data](#sensitive-data) guide) [`TD1` and `TD1-ON` forms](https://www.canada.ca/en/revenue-agency/services/forms-publications/td1-personal-tax-credits-returns.html) to [Drive](https://link.hypha.coop/drive). 
+
+1. Add Member [as Employee in Wagepoint](#adding-a-new-employee) for payroll.
+
+## Payroll
+
+This guide describes how to use our payroll service provider, [Wagepoint](https://wagepoint.com), to set up a payroll run for the pay period.
+You should have a completed [Employee Payroll](http://link.hypha.coop/payroll) sheet for the pay period ready, which indicates the wage information for each Employee that you will need to enter into Wagepoint.
+
+If this is the first time you use Wagepoint, please first review the [Running your first payroll with Wagepoint - Canada](https://www.youtube.com/watch?v=KfLMcqGTDwE)video.
+
+### Running payroll
+
+1. Log into Wagepoint as a user with admin privileges.
+
+1. Visit the `PAYROLL` tab.
+
+1. **Paygroup:** Hit `NEXT` since we only have one monthly paygroup.
+
+    ![payroll-running-payroll-0](images/payroll-running-payroll-0.png)
+
+1. **Paydates:** Set the pay cycle to cover the current month (e.g., March 1-31) and the `Pay Date` to be the 15th of the current month (e.g., March 15) or the last working day prior if the 15th happens to land on a holiday (e.g., March 13).
+    During that period, we settle amounts owed up to the _end of the previous month_ (e.g., Feb 29).
+
+    ![payroll-running-payroll-1](images/payroll-running-payroll-1.png)
+    
+    Set the dates as per the above, and hit `NEXT`.
+    Nothing is finalized until the last step, and you must hit `SAVE/NEXT` during each step to preserve "draft" progress.
+
+1. **Hours:** Hit `SAVE/NEXT` to skip since we currently do not use the `Hourly` pay type.
+
+1. **Salary:** Enter `HOURS`, `CURRENT PAY`, and `PUBLIC HOLIDAY PAY` according to the [Employee Payroll](http://link.hypha.coop/payroll) sheet for the pay period.
+    Ensure that hours and pay line up, as these are the basis for ensuring legal minimum wage.
+
+    The `VACATION HRS` for everyone is `0` due to our variable work hours and that annualized amounts will be paid out each pay period.
+    
+    In the `PAY?` column, select `YES` for any Employee who will be paid in this payroll.
+
+    You can review past payrolls under the `REPORTS` tab.
+
+1. **Process:** Verify the amounts and `APPROVE PAYROLL`. 
+
+1. **Confirm:** Confirm the payroll run for this pay period.
+    It will be queued for processing on the `Pay Date`.
+
+1. Save the PDF output of `VIEW ALL`, our "Payroll Register", into the 🔒 [`Finance > Payroll` folder in our Drive](https://drive.google.com/drive/u/0/folders/1IWyDzyeixO4Z6TbIghorqTiCbrs2N4K8).
```

Comment:
```suggestion
1. **Process:** Verify the amounts.
    Save the PDF output of `VIEW ALL`, our "Payroll Register", into the 🔒 [`Finance > Payroll` folder in our Drive](https://drive.google.com/drive/u/0/folders/1IWyDzyeixO4Z6TbIghorqTiCbrs2N4K8).
    Click `APPROVE PAYROLL`.

1. **Confirm:** Confirm the payroll run for this pay period.
    It will be queued for processing on the `Pay Date`.
```